src/components/Search/SearchResults/SearchResultItem.tsx

```diff
@@ -17,10 +17,10 @@ const SearchResultItem: React.FC<Props> = ({ result }) => (
         <div className={styles.quranTextContainer}>
           <p className={styles.verseKey}>{result.verseKey}</p>
           <div className={styles.quranTextResult}>
-            {result.words.map((word) => (
+            {result.words.map((word, index) => (
               <QuranWord
                 highlight={!!word.highlight}
-                key={word.id}
+                key={`${result.verseKey}:${index + 1}`}
```

Comment:
does word.id return a backend related identifier?

Reply:
BE just returns: `char_type_name` and `text` for each word without any unique identifiers so have to manually construct the words `location` which is unique.